src/engine/controls/cuecontrol.cpp

```diff
@@ -85,7 +85,11 @@ CueControl::CueControl(const QString& group,
           m_bypassCueSetByPlay(false),
           m_iNumHotCues(NUM_HOT_CUES),
           m_pLoadedTrack(),
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+          m_mutex() {
```

Comment:
No need to explicitly invoke the default constructor. Also applies to `m_pLoadedTrack()` above.

```
#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
{
#else
, m_mutex(QMutex::Recursive) {
#endif
```